src/xrAICore/Navigation/level_graph_inline.h

```diff
@@ -571,6 +571,14 @@ IC void CLevelGraph::clear_mask_no_check(u32 vertex_id) { m_access_mask[vertex_i
 template <typename P>
 IC void CLevelGraph::iterate_vertices(
     const Fvector& min_position, const Fvector& max_position, const P& predicate) const
+{
+    auto [I, E] = get_range(min_position, max_position);
```

Comment:
Although there's `I` and `E` in the codebase everywhere (original code), we don't use that with the current codestyle.
```suggestion
    auto [it, ite] = get_range(min_position, max_position);
```